src/Components/Components/src/ComponentBase.cs

```diff
@@ -175,6 +175,14 @@ protected Task InvokeAsync(Action workItem)
     protected Task InvokeAsync(Func<Task> workItem)
         => _renderHandle.Dispatcher.InvokeAsync(workItem);
 
+    /// <summary>
+    /// Dispatches an <see cref="Exception"/> to the renderer via the <see cref="RenderHandle"/>.
```

Comment:
Suggestion fr giving more info about the effect of the call:

```suggestion
    /// Treats the supplied <paramref name="exception"/> as being thrown by this component. This will cause the
    /// enclosing <see cref="ErrorBoundary"/> to transition into a failed state. If there is no enclosing <see cref="ErrorBoundary"/>,
    /// it will be regarded as an exception from the enclosing renderer.
    ///
    /// This is useful if an exception occurs outside the component lifecycle methods, but you wish to treat it
    /// the same as an exception from a component lifecycle method.
```

Reply:
I only suggest giving all this detail here because `ComponentBase` is a super common API to be using.

The XML docs on `RenderHandle` below already look ideal to me, as that's a more low-level and advanced part of the API surface.

Reply:
I changed the `<see />` tags since `ComponentBase` doesn't know what `ErrorBoundary` is.

Reply:
Ah yes, that's because it's in `M.A.Components.Web`. The way you've resolved this looks great.